cmd/generate/generate.go

```diff
@@ -124,17 +124,18 @@ func Tag(params Params, gc gitClient) (Result, error) {
 	}
 
 	var (
-		finalTag        string
-		ancestorTag     string
-		previousPattern string
-		isPrerelease    bool
+		finalTag       string
+		ancestorTag    string
+		includePattern string
+		excludePattern string = ""
```

Comment:
This is not needed, as empty string is the nil type of string.
```suggestion
		excludePattern string
```